### ISSUE TITLE
Recent Documents

### ISSUE THREAD

**Issue by slipttees** [lxqt/lxqt#2118]
It could have an option in the Pcmanfm Menu>Files>Recent Documents for a list of opening recent files.

This was a big help to my dad and mom use computer for basic use when they used gnome-classic.

I miss that!

Image for example!
![unnamed](https://user-images.githubusercontent.com/25433004/140982663-e07cfa35-18d8-42e5-a723-ae006bc13bd7.png)

Bye folks

**Comment by tsujan** [comment]
It's possible, with "disabled" as the default. But it should be popular enough; otherwise, the code that needs to be added wouldn't be justified. We haven't received any feature request about it other than this one and I don't think it exists in other file managers.

**Comment by tsujan** [comment]
Oh, I think what you see in gnome-classic is just `recent:///`. pcmanfm-qt already supports it (provided that you have GVFS). Just type it in its path bar.

**Comment by slipttees** [comment]
This is not friendly!

**Comment by tsujan** [comment]
> This is not friendly!

What is not friendly?

**Comment by slipttees** [comment]
> EDIT: Its contents can't be opened directly by Qt apps; you need to copy-paste them somewhere else.

Here

**Comment by tsujan** [comment]
No, it isn't. The reason is that it's only for GTK apps and LXQt is a Qt DE. That's why we haven't added it to the left pane.

It's still possible to implement "Recent Files" directly, without `recent:///`. Let's wait for other comments.

**Comment by tsujan** [comment]
I transfer this report to pcmanfm-qt's repository.
